app/client/cli/actor.go

```diff
@@ -0,0 +1,151 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pokt-network/pocket/utility/types"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(NewActorCommands()...)
+}
+
+type actorCmdDef struct {
+	Name      string
+	ActorType types.ActorType
+}
+
+func NewActorCommands() (cmds []*cobra.Command) {
+
+	actorCmdDefs := []actorCmdDef{
+		{"Application", types.ActorType_App},
+		{"Node", types.ActorType_Node},
+		{"Fisherman", types.ActorType_Fish},
+		{"Validator", types.ActorType_Val},
+	}
+
+	for _, cmdDef := range actorCmdDefs {
+
+		cmd := &cobra.Command{
+			Use:     cmdDef.Name,
+			Short:   fmt.Sprintf("%s actor specific commands", cmdDef.Name),
+			Aliases: []string{strings.ToLower(cmdDef.Name), cmdDef.ActorType.GetActorName()},
+			Args:    cobra.ExactArgs(0),
+		}
+		cmd.AddCommand(newActorCommands(cmdDef)...)
+		cmds = append(cmds, cmd)
+	}
+
+	return cmds
+}
+
+func newActorCommands(cmdDef actorCmdDef) (cmds []*cobra.Command) {
+
+	stakeCmd := &cobra.Command{
+		Use:   "Stake <from> <amount>",
+		Short: "Stake <from> <amount>",
+		Long:  fmt.Sprintf(`Stakes <amount> for the %s actot with address <from>`, cmdDef.Name),
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// TODO(deblasis): update when we have keybase
+			pk, err := readEd25519PrivateKeyFromFile(privateKeyFilePath)
+			if err != nil {
+				return err
+			}
+			// currently ignored since we are using the address from the PrivateKey
+			// fromAddr := crypto.AddressFromString(args[0])
+			amount := args[1]
+
+			_ = &types.MessageStake{
+				PublicKey:     pk.PublicKey().Bytes(),
+				Chains:        []string{}, // TODO(deblasis): 👀
```

Comment:
This is still in flux, but we will probably either end up having something similar to v0:

<img width="944" alt="Screen Shot 2022-08-23 at 7 21 28 PM" src="https://user-images.githubusercontent.com/1892194/186282833-8c8f7963-4642-4d58-b9e5-706592f4e59c.png">

Or be able to pass in a path to a config to make it easier:

<img width="695" alt="Screen Shot 2022-08-23 at 7 18 56 PM" src="https://user-images.githubusercontent.com/1892194/186282726-823432ae-08b8-4caa-b462-0d2e9ba29a6a.png">

Lets leave as is for now and cc @andrewnguyen22 for context